src/core/ugbio_core/trimmer_utils.py

```diff
@@ -95,6 +96,23 @@ def read_trimmer_failure_codes(
         columns={c: c.replace(" ", "_").lower() for c in df_trimmer_failure_codes.columns}
     )
 
+    # If there's more than one format, keep the failed_read_count from the last format rows,
+    # and take total_read_count from the first (per [read group, segment, reason] combination)
+    if not (df_trimmer_failure_codes["format"] == df_trimmer_failure_codes["format"].iloc[0]).all():
```

Comment:
```suggestion
    if df_trimmer_failure_codes["format"].nunique()>1:
```
I think that's cleaner